MagickCore/compress.h

```diff
@@ -43,6 +43,8 @@ typedef enum
   NoCompression,
   PizCompression,
   Pxr24Compression,
+  DwaaCompression,
```

Comment:
These values should be below `WebPCompression` as the last two values. This should be done because we don't want to change the numeric value of the current values.